packages/localize/src/date/getMonthNames.js

```diff
@@ -2,24 +2,31 @@ import { normalizeDate } from './normalizeDate.js';
 
 const monthsLocaleCache = {};
 
-export function getMonthNames({ locale }) {
-  let months = monthsLocaleCache[locale];
+/**
+ * @desc Returns month names for locale
+ * @param {string} options.locale locale
+ * @param {string} [options.style=long] long, short or narrow
+ * @returns {Array} like: ['Januray', 'February', ...etc].
+ */
+export function getMonthNames({ locale, style = 'long' } = {}) {
```

Comment:
is there also a `narrow` variation?

Reply:
Yes: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat